packages/client/src/components/Svg.js

```diff
@@ -168,3 +168,29 @@ export const Check = ({ width = "24", height = "24", className = "" }) => (
     <path stroke="#fff" d="M5 13l5 4 8-9" />
   </svg>
 );
+
+export const CoinIcon = ({ coinType = "FLOW", width = "24", height = "24" }) => {
+  if (coinType == COIN_TYPES.FLOW) {
+    return (
+      <svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M12 24C18.6274 24 24 18.6274 24 12C24 5.37258 18.6274 0 12 0C5.37258 0 0 5.37258 0 12C0 18.6274 5.37258 24 12 24ZM17.2561 10.1285H13.8721V13.5125L10.4885 13.5125V13.5124H10.4878V10.1285H9.21655C6.64855 10.1285 4.56055 12.2165 4.56055 14.7845C4.56055 17.3525 6.64855 19.4405 9.21655 19.4405C11.7845 19.4405 13.8725 17.3525 13.8725 14.7845V13.5125L17.2561 13.5125V10.1285ZM10.4885 13.5125V14.7845C10.4885 15.4805 9.91255 16.0565 9.21655 16.0565C8.52055 16.0565 7.94455 15.4805 7.94455 14.7845C7.94455 14.0885 8.52055 13.5125 9.21655 13.5125H10.4885ZM15.1438 8.42426H18.9598V5.04026H15.1438C12.5758 5.04026 10.4878 7.12826 10.4878 9.69626V10.1283H13.8718V9.69626C13.8718 9.00026 14.4478 8.42426 15.1438 8.42426Z"
+          fill="#000E64"
+        />
+      </svg>
+    );
+  } else {
+    return (
+      <svg width={width} height={height} viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
```

Comment:
I think every token supposed to have their own icon, @andrewnedimyer can we have a their name mapping on figma too  https://www.figma.com/file/dDUffk0WfPkUjPlmAj2dlt/Vessel%3A-Design-Library?node-id=26%3A1514

Reply:
I see we are lacking icons for FUSD, BLT, STARLY and REVV @andrewnedimyer

Reply:
@linxiao-li @andrewnedimyer
Added new icons to figma here: https://www.figma.com/file/sKtcz6ytBmNNDNW7XU8ea5/Vessel%3A-User-Experience?node-id=572%3A135576

Also exported all icons (including tokens) as png/svg here: https://drive.google.com/drive/folders/1Xoj8-fMep4jHEt16_4YvFIsRJLqCqktG?usp=sharing

Also

Reply:
@linxiao-li  would you be able to add those SVGs for each token in this PR? if not, we can create a ticket to track that work.

Reply:
@amyliu6 created a ticket for the icon work: https://linear.app/dappercollectives/issue/VSL-183